docs/shared-content-source/docs/modules/administration/pages/how-to-install-and-use-strimzi.adoc

```diff
@@ -0,0 +1,136 @@
+
+
+include::ROOT:partial$include.adoc[]
+
+This guide shows how to install Strimzi and create a Kafka cluster suitable for use with Cloudflow.
+
+Strimzi is a Kubernetes operator that simplifies the process of running Apache Kafka.
+
+== Prerequisites
+
+=== CLI Tools
+
+Make sure you have the following prerequisites installed before continuing:
+
+- Helm, version 3 or later
+- Kubectl 
+
+== Installing Strimzi
+
+In this guide, we will use Helm to install Strimzi.
+
+We are going to create Strimzi in the `cloudflow` namespace in this guide. Make sure that the namespace `cloudflow` exists before continuing. To create the `cloudflow` namespace, execute
+
+  kubectl create ns cloudflow
+
+Add the Strimzi Helm repository and update the local index.
+
+  helm repo add strimzi https://strimzi.io/charts/
+  helm repo update
+
+Install the latest version of Strimzi.
+
+  helm install strimzi strimzi/strimzi-kafka-operator --namespace cloudflow
+
+After the install complete, the Strimzi Kafka operator should be running in the `cloudflow` namespace.
+
+----
+$ kubectl get pods -n cloudflow
+NAME                                       READY   STATUS    RESTARTS   AGE
+strimzi-cluster-operator-9968fd8c9-fhqmj   1/1     Running   0          17s
+----
+
+=== Creating a Kafka cluster using Strimzi
+
+To create a Kafka cluster using Strimzi, we have to create a `CustomResource` of the kind `Kafka` in the namespace `cloudflow`. 
+
+NOTE: The Kafka cluster configuration shown here is meant for development and testing purposes only. Please consider your storage requirements and modify the custom resource accordingly for your intended usage.
+
+You can use the following command to create a cluster:
+
+[source,shell script]
+----
```

Comment:
also the files ideally should come from:
https://github.com/lightbend/cloudflow/blob/master/integration-tests/kafka-cluster.yaml

Reply:
I could imagine that the integration tests would do slightly different things. Also, I think with antora the snippets need to be in a specific place, sadly

Reply:
yes, examples need to be in examples/doc-snippets. However, a soon-coming release of antora should remove this limitation.

Reply:
please keep us posted when the limitation is removed!